_pages/install/browser-support.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Web Browser Support
+---
+
+Spartacus libraries are designed to take advantage of the latest web platform standards, while also allowing you to run your Spartacus storefront on as many different web browsers as possible. However, some older browsers do not support the latest standards, and as a result, these browsers are not supported by Spartacus. Officially, Spartacus only supports evergreen browsers.
+
+However, there are ways to get Spartacus to work with older browsers, including IE11 (even though it is not an evergreen browser). The following are some of the steps you can take to get older browsers to work with Spartacus:
+
+- You can transpile TypeScript into other version of JavaScript
+- You can use polyfills to provide some of the standard web features that are missing in some older browsers
+- You can use PostCSS to automatically add vendor prefix style rules.
+
+## Working with IE11
+
+If your Spartacus storefront needs to support IE11, you must do the following:
+
+- Use JavaScript polyfills to add missing browser capabilities
+- Customize the Spartacus styles, or bring in a custom style layer.
```

Comment:
Perhaps some more explanations here: 
The default style library of Spartacus is not prepared for IE11. Since a large portion of customers will not use these styles in the first place, it's a natural custom effort to build your styles in such a way that it supports the required browsers. 

Some of the standard CSS features used in Spartacus styles that won't work for IE11:
- custom properties are used in Spartacus themes and are not supported in IE11. See https://caniuse.com/?search=custom%20properties.
- logical properties are used for [bidirectional text and layout](https://sap.github.io/spartacus-docs/directionality/), which is not supported in IE11. See https://caniuse.com/?search=logical%20properties.

Reply:
Done 🙂